apps/financial-aid/backend/src/app/modules/application/index.ts

```diff
@@ -1,3 +1,4 @@
 export { ApplicationModel } from './models'
 export { ApplicationService } from './application.service'
 export { ApplicationModule } from './application.module'
+export { ApplicationController } from './application.controller'
```

Comment:
Should we remove this, especially since we talked about IRL removing index files in the BE